azure-stack/hci/concepts/azure-stack-hci-dashboard.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Azure Stack HCI dashboard
+description: Learn to view and navigate your Azure Stack HCI dashboard in Azure portal.
+ms.topic: conceptual
+author: alkohli
+ms.subservice: azure-stack-hci
+ms.author: alkohli
+ms.reviewer: alkohli
+ms.date: 02/13/2024
+---
+
+# Azure Stack HCI dashboard
+
+[!INCLUDE [hci-applies-to-23h2](../../includes/hci-applies-to-23h2.md)]
+
+This article details how to view and navigate your Azure Stack HCI dashboard in Azure portal. You can display the dashboard in chart view or in list view.
+
+## View the dashboard chart view
+
+The Azure Stack HCI dashboard displays overview and detailed information about your clusters.
+
+To view the dashboard chart view:
+
+1. In Azure portal, go to **Azure Arc** > **Infrastructure** > **Azure Stack HCI** > **Overview (preview)**.
+
+   You can filter the dashboard display by Subscription, Resource group, and Region.
+
```

Comment:
Change to: You can filter the dashboard display by **Subscription**, **Resource group**, and **Region**.

Reply:
Done > revised to bold.